SWLOR.Game.Server/Service/Race.cs

```diff
@@ -90,6 +90,12 @@ public static void LoadRaces()
                 AppearanceType = AppearanceType.HalfOrc,
                 HeadId = 27
             };
+            _defaultRaceAppearancesMale[RacialType.Togruta] = new RacialAppearance
```

Comment:
Togruta and Rodian are duplicated twice with different values. Can you remove the ones that shouldn't be here?

Reply:
Disregard, I forgot this was by gender. My bad.